tools/accuracy_checker/accuracy_checker/metrics/detection.py

```diff
@@ -618,3 +618,87 @@ def _prepare_prediction_boxes(label, predictions, ignore_difficult):
 
 def get_valid_labels(labels, background):
     return list(filter(lambda label: label != background, labels))
+
+class YoutubeFacesAccuracy(FullDatasetEvaluationMetric):
+
+    __provider__ = 'youtube_faces_accuracy'
+    annotation_types = (DetectionAnnotation, )
+    prediction_types = (DetectionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'overlap': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.40, optional=True,
+                description='Specifies the IOU threshold to consider as a true positive candidate face.'
+            ),
+            'relative_size': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.25, optional=True,
+                description='Specifies the size of detected face candidate\'s area in proportion to the size '
+                'of ground truth\'s face size. This value is set to filter candidates that have high IOU '
+                'but have a relatively smaller face size than ground truth face size.'
+            )
+        })
+        return parameters
+
+    def configure(self):
+        self.overlap = self.get_value_from_config('overlap')
+        self.relative_size = self.get_value_from_config('relative_size')
+
+    def submit_all(self, annotations, predictions):
+        return self.evaluate(annotations, predictions)
+
+    def evaluate(self, annotations, predictions):
+        true_positive = 0
+        false_positive = 0
+
+        for (annotation, prediction) in zip(annotations, predictions):
```

Comment:
why does this process done in update method?
You evaluate image by image, so you can logic of image processing  and increment tp and fp to update, it allows you to save time on final evaluation

Reply:
ok, lets keep as is. I possibly refactor it later if we will need to quantize with Accuracy Aware algo